Umee-Bots/uToken-Exchange-Rate-Drop/src/utils.ts

```diff
@@ -0,0 +1,88 @@
+import { Finding, FindingSeverity, FindingType, ethers } from "forta-agent";
+import { Interface } from "ethers/lib/utils";
+import BigNumber from "bignumber.js";
+
+export interface AgentConfig {
+  uTokens: Array<{ uToken: string; address: string }>;
+  uTokenPairs: Array<{ uToken1: string; uToken2: string; threshold: number; difference: number }>;
+  umeeOracle: string;
+  lendingPool: string;
+}
+
+export const UTOKENS_IFACE = new Interface(["function UNDERLYING_ASSET_ADDRESS() public view returns (address)"]);
+
+export const UMEE_ORACLE_PRICE_IFACE = new Interface([
+  "function getAssetPrice(address asset) external view returns (uint256)",
+]);
+
+export const LENDING_POOL_IFACE = new Interface([
+  "function getReserveNormalizedIncome(address asset) public view returns (uint256)",
+]);
+
+export function createFinding(
+  pair: string,
+  previousRatio: BigNumber,
+  currentRatio: BigNumber,
+  severity: FindingSeverity
+): Finding {
+  return Finding.from({
+    alertId: "UMEE-2",
+    name: "uToken exchange ratio drop",
+    description: `uToken pair of ${pair} has dropped.`,

```

Comment:
```suggestion
    description: `${pair} rate had a significant drop from the previous block`,
```
Wdyt about something like this? Feel free to edit.

Reply:
alright. more descriptive 👍